src/controllers/objecting.entity.name.controller.ts

```diff
@@ -2,17 +2,20 @@ import { NextFunction, Request, Response } from "express";
 import { check, Result, ValidationError, validationResult } from "express-validator";
 import { ErrorMessages } from "../model/error.messages";
 import { createGovUkErrorData, GovUkErrorData } from "../model/govuk.error.data";
-import { OBJECTIONS_COMPANY_NUMBER } from "../model/page.urls";
+import { OBJECTIONS_CHECK_YOUR_ANSWERS, OBJECTIONS_COMPANY_NUMBER } from "../model/page.urls";
 import {
-  addObjectionCreateToObjectionSession, retrieveFromObjectionSession,
-  retrieveObjectionSessionFromSession
+  addObjectionCreateToObjectionSession,
+  retrieveAccessTokenFromSession,
+  retrieveCompanyProfileFromObjectionSession,
+  retrieveFromObjectionSession,
 } from "../services/objection.session.service";
 import { Objection, ObjectionCreate } from "../modules/sdk/objections";
 import { Session } from "ch-node-session-handler";
 import { Templates } from "../model/template.paths";
-import { getObjection } from "../services/objection.service";
+import {getObjection, updateObjectionReason, updateObjectionUserDetails} from "../services/objection.service";
```

Comment:
spaces

Reply:
Put spaces in.

Reply:
spaces still missing for me, i think your last commit was just the unit test

Reply:
the updateObjectionReason is showing as an unused import in GoLand. You can check for errors in your IDE before committing, they will be highlighted in different colours to make them stand out. We should also have eslint rules for unused imports and bracket spaces.